demisto_sdk/commands/pre_commit/hooks/hook.py

```diff
@@ -14,20 +19,21 @@ def __init__(
         self,
         hook: dict,
         repo: dict,
-        mode: str = "",
-        all_files: bool = False,
-        input_mode: bool = False,
+        context: "PreCommitContext",
```

Comment:
because of type checking, but now when i moved it to a different file we can remove this

Reply:
It currently has to be with quoutes since the context needs `Hook.get_property` to remove the hooks with `skip` property or remove hooks which `needs` a hook that is skipped.

Reply:
fixed